include/btllib/aahash_consts.hpp

```diff
@@ -0,0 +1,2582 @@
+#ifndef BTLLIB_AAHASH_CONSTS_HPP
+#define BTLLIB_AAHASH_CONSTS_HPP
+
+#include <cmath>
+#include <cstdint>
+#include <vector>
+
+#include "btllib/nthash_consts.hpp"
+// clang-format off
+namespace btllib {
+
+#define AA_ROLL_TABLE(CHAR, LEVEL, ROT)                                        \
+  (LEVEL_X_AA_SEED_LEFT_31BITS_ROLL_TABLE[LEVEL][CHAR]                         \
+                                         [(ROT) < 31 ? (ROT) : (ROT) % 31] |   \
+   LEVEL_X_AA_SEED_RIGHT_33BITS_ROLL_TABLE[LEVEL][CHAR]                        \
+                                          [(ROT) < 33 ? (ROT) : (ROT) % 33])
+
+// 64-bit random seeds corresponding to 20 amino acids
+static const uint64_t AA_SEED_A = 0xf56d6192468323df;
```

Comment:
`static` shouldn't be in a `.hpp` file. Moreover, I'd declare this (and all other constants, including arrays) as `constexpr inline` which more accurately conveys its meaning, as is done in standard library too: https://en.cppreference.com/w/cpp/header/numbers

Reply:
done